### PR TITLE
[Accuracy diff No.112] Fix accuracy diff for paddle.nn.functional.conv1d API

### PR DESCRIPTION
**Pull request by co63oc**
配置report/ci_ce_cpu/error_config.txt
PaddleAPITest CPU测试为torch error, weight类型和输入类型不一致
image

配置report/ci_ce_cpu/20250529/5_accuracy/error_config.txt中测试通过
image

### CODE REVIEW

**Comment by cangtianhuang** [comment]
@co63oc 如果是 torch error，可以尝试修改 tester/paddle_to_torch/rules.py 中的 Conv1dRule，手动将 input 和 weight 的类型对齐，使其支持精度测试😉~

**Comment by cangtianhuang** [comment]
这个看上去是一个 amp 配置，测试时请添加 --test_amp=True

在 CPU 测试 --test_cpu=True 会报 paddle error，这是我们应该修复的：
image

**Comment by cangtianhuang** [comment]
> > @co63oc 如果是 torch error，可以尝试修改 tester/paddle_to_torch/rules.py 中的 Conv1dRule，手动将 input 和 weight 的类型对齐，使其支持精度测试😉~
> 
> 尝试修改rules.py和accuracy.py，需要考虑paddle, torch都要转换类型，反向也要转换类型，改动较多
> 
> 更改了测试用例中类型参数，测试可以通过，不确定这样是否可以 paddle.nn.functional.conv1d(Tensor([8, 400, 100],"float32"), Tensor([256, 100, 3],"float32"), bias=Tensor([256],"float32"), padding=1, stride=list[1,], dilation=list[1,], groups=4, data_format="NCL", )
> 
> ```
> python engine.py --accuracy=True --api_config='paddle.nn.functional.conv1d(Tensor([8, 400, 100],"float32"), Tensor([256, 100, 3],"float32"), bias=Tensor([256],"float32"), padding=1, stride=list[1,], dilation=list[1,], groups=4, data_format="NCL", )'
> ```
> 
> image

此配置 torch 能够通过，说明是有效配置，不用修改配置~

**Comment by co63oc** [comment]
https://github.com/PaddlePaddle/Paddle/pull/73535 已在paddle提交PR

**Comment by co63oc** [comment]
https://github.com/PaddlePaddle/Paddle/pull/73535 已合入，关闭当前PR
